### ISSUE TITLE
[Bug]: Name Error for LHFLX in Tables

### ISSUE THREAD

**Issue by kchong75** [E3SM-Project/e3sm_diags#818]
### What happened?

Hi there, I noticed an inconsistency in the table description for the variable LHFLX. It is currently shown as "Surface water flux", but it should be "Surface latent heat flux". 

Please refer to this screenshot:
<img width="1171" alt="image" src="https://github.com/E3SM-Project/e3sm_diags/assets/48594631/7975d562-2965-49f9-9ea2-9ba1e9625fc0">

Could this be because LHFLX is calculated from QFLX, even though the variable LHFLX is already present in my input file?

Thank you for looking into this.

### Environment

```
     active environment : e3sm_unified_1.10.0_login
    active env location : /global/common/software/e3sm/anaconda_envs/base/envs/e3sm_unified_1.10.0_login
            shell level : 2
       user config file : /global/homes/k/kchong37/.condarc
 populated config files : 
          conda version : 24.5.0
    conda-build version : 24.3.0
         python version : 3.8.18.final.0
                 solver : libmamba (default)
       virtual packages : __archspec=1=zen3
                          __conda=24.5.0=0
                          __cuda=12.2=0
                          __glibc=2.31=0
                          __linux=5.14.21=0
                          __unix=0=0
       base environment : /global/common/software/e3sm/anaconda_envs/base  (read only)
      conda av data dir : /global/common/software/e3sm/anaconda_envs/base/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /global/common/software/e3sm/anaconda_envs/base/pkgs
                          /global/homes/k/kchong37/.conda/pkgs
       envs directories : /global/common/software/nersc/pe/conda-envs/24.1.0/python-3.11
                          /global/homes/k/kchong37/.conda/envs
                          /global/common/software/e3sm/anaconda_envs/base/envs
               platform : linux-64
             user-agent : conda/24.5.0 requests/2.31.0 CPython/3.8.18 Linux/5.14.21-150400.24.81_12.0.87-cray_shasta_c sles/15.4 glibc/2.31 solver/libmamba conda-libmamba-solver/24.1.0 libmambapy/1.5.8
                UID:GID : 104008:104008
             netrc file : None
           offline mode : False

```

**Comment by chengzhuzhang** [comment]
@kchong75 Kezhen Thank you for reporting this problem. Yes, you are right, the name of the variable is not updated in this case after QFLX is converted to LHFLX. A variable name change is needed in following code:
https://github.com/E3SM-Project/e3sm_diags/blob/c8404ab0f5523fee80cda396357f307b7ad1ff5b/e3sm_diags/derivations/acme.py#L210-L215
I will issue a PR to resolve.

**Comment by tomvothecoder** [comment]
Closed by #820 and #876
